TACC/protx-dashboard
# task/COOKS-293:  add DFPS regions

## Overview: ##

Add DFPS regions to demographics section. This PR works with https://github.com/TACC/protx-geospatial-data/pull/3

* add dfps regions
* fix downloading of census tracts
* update dropdown of area selection (i.e. COOKS-296)
* fixes issue when switching geographic areas but an area was already selected (i.e. COOKS-298)

## Related Jira tickets: ##

* [COOKS-293](https://jira.tacc.utexas.edu/browse/COOKS-293)
* [COOKS-296](https://jira.tacc.utexas.edu/browse/COOKS-296)
* [COOKS-298](https://jira.tacc.utexas.edu/browse/COOKS-298)

## Testing Steps: ##

COOKS 293
1. ~Ensure that either https://github.com/TACC/protx-dashboard/pull/23 is merged or in this branch or that you do a `docker-compose down -v` before you start working~. (update: I've merged #23 into this branch) Set up as usual by using the README and docker-compose up
2. Go to https://cep.dev/protx/dash/demographics
3. Select DFPS Regions
4. Check that different demographics works
5. Check that you can zoom in and see chart.
6. Check that you can download resources for the selected DFPS Region
7.  Also ensure that you can download resources for selected counties.

COOKS-296
1. Check that geography selection matches the JIRA ticket


COOKS-298
1. Select a county.  See chart
2. Select census tract or dfps and ensure that county selection is reset.

 
## Notes: ##

TODO:

- [x] ensure protx-geospatial-data image is pushed
- [x] check charts
- [x] check downloading of resources
